workflows/genomeassembly.nf

```diff
@@ -35,21 +40,26 @@ if (params.organelles_on) { organelles_on = params.organelles_on } else { organe
 //
 // SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
 //
-include { PREPARE_INPUT                                    } from '../subworkflows/local/prepare_input'
-include { RAW_ASSEMBLY                                     } from '../subworkflows/local/raw_assembly' 
-include { ORGANELLES                                       } from '../subworkflows/local/organelles' 
-include { GENOMESCOPE_MODEL                                } from '../subworkflows/local/genomescope_model'
-include { PURGE_DUPS                                       } from '../subworkflows/local/purge_dups'
-include { POLISHING                                        } from '../subworkflows/local/polishing'
-include { SCAFFOLDING                                      } from '../subworkflows/local/scaffolding'
-include { KEEP_SEQNAMES as KEEP_SEQNAMES_PRIMARY           } from '../modules/local/keep_seqnames'
-include { KEEP_SEQNAMES as KEEP_SEQNAMES_HAPLOTIGS         } from '../modules/local/keep_seqnames'
-include { HIC_MAPPING                                      } from '../subworkflows/local/hic_mapping'
-include { GENOME_STATISTICS as GENOME_STATISTICS_RAW       } from '../subworkflows/local/genome_statistics'
-include { GENOME_STATISTICS as GENOME_STATISTICS_RAW_HIC   } from '../subworkflows/local/genome_statistics'
-include { GENOME_STATISTICS as GENOME_STATISTICS_PURGED    } from '../subworkflows/local/genome_statistics'
-include { GENOME_STATISTICS as GENOME_STATISTICS_POLISHED  } from '../subworkflows/local/genome_statistics'
-include { GENOME_STATISTICS as GENOME_STATISTICS_SCAFFOLDS } from '../subworkflows/local/genome_statistics'
+include { PREPARE_INPUT                                         } from '../subworkflows/local/prepare_input'
+include { RAW_ASSEMBLY                                          } from '../subworkflows/local/raw_assembly' 
+include { ORGANELLES                                            } from '../subworkflows/local/organelles' 
+include { GENOMESCOPE_MODEL                                     } from '../subworkflows/local/genomescope_model'
+include { PURGE_DUPS                                            } from '../subworkflows/local/purge_dups'
+include { POLISHING                                             } from '../subworkflows/local/polishing'
+include { SCAFFOLDING                                           } from '../subworkflows/local/scaffolding'
+include { SCAFFOLDING as SCAFFOLDING_HAP1                       } from '../subworkflows/local/scaffolding'
+include { SCAFFOLDING as SCAFFOLDING_HAP2                       } from '../subworkflows/local/scaffolding'
```

Comment:
Instead of using 1 or 2 as suffix, can you do similar like others, appended with the actual function?

Reply:
I saw others use the word instead of number. Then it is fine.